source/hp/dof_handler.cc

```diff
@@ -609,7 +617,7 @@ namespace internal
               types::global_dof_index cache_size = 0;
               for (typename HpDoFHandler<dim,spacedim>::active_cell_iterator
                    cell=dof_handler.begin_active(level);
-                   cell!=dof_handler.end_active(level); ++cell)
+                   cell!=end_active_cache[level]; ++cell)
```

Comment:
I agree with @bangerth here in that we typically use the `endc` variable at places like this.